packages/bpk-component-calendar/src/BpkCalendarGridHeader.js

```diff
@@ -54,24 +60,30 @@ const WeekDay = props => {
 
   return (
     <Element className={classNames.join(' ')} title={weekDay.name}>
-      <span aria-hidden="true">{weekDay.nameAbbr}</span>
+      <span aria-hidden="true">{weekDay[weekDayKey || 'nameAbbr']}</span>
```

Comment:
nit: `|| 'nameAbbr'` shouldn't be necessary anymore as this is the default prop

Reply:
Fixed